src/Algebra/Properties/AbelianGroup.agda

```diff
@@ -36,3 +36,13 @@ xyx⁻¹≈y x y = begin
   x ⁻¹ ∙ y ⁻¹ ≈⟨ ⁻¹-anti-homo-∙ y x ⟨
   (y ∙ x) ⁻¹  ≈⟨ ⁻¹-cong $ comm y x ⟩
   (x ∙ y) ⁻¹  ∎
+
+module _ where
+  open IsGroup isGroup using (_//_; _\\_)
+
+  \\≗//ᵒ : ∀ x y → x \\ y ≈ y // x
```

Comment:
If we have `\\≗//⇒comm`, I feel like this should be `comm⇒\\≗//` in `Properties.Group` and then appeal to that here? Also the unexplained `ᵒ` in the name again could be dropped?